samples/unit-tests/axis/stacklabels/demo.js

```diff
@@ -21,27 +21,26 @@ QUnit.test('Stack labels on non-data axis', function (assert) {
     });
 
 
-    assert.close(
+    assert.strictEqual(
         chart.container.querySelector('.highcharts-stack-labels text')
-            .getBBox().y,
-        0,
-        1,
-        'The y attribute is around to 0.'
+            .getAttribute('y') !== null,
+        true,
+        'Y attribute is set (#8834)'
```

Comment:
Instead of stating what it is (which is ambiguous when the test fails), say what it _should_ be. ("The Y attribute should be set (#8834)").